clio-server/src/main/resources/reference.conf

```diff
@@ -8,14 +8,16 @@
 include "clio-server-version"
 
 clio.server {
-  environment: dev
-
   http-server {
     interface: 0.0.0.0
     port: 8080
     shutdown-timeout: 60s
   }
 
+  persistence {
+    service-account-json: null
```

Comment:
does ficus turn this in to a null Path or a Path to "null"?

Reply:
The `getAs[Path]` method turns it into a `None` if null, or `Some[Path]` if given.